packages/compass-aggregations/src/modules/update-view.js

```diff
@@ -91,22 +91,13 @@ export const updateView = () => {
           num_stages: viewPipeline.length,
           editor_view_type: mapPipelineModeToEditorViewType(state),
         });
+        debug('selecting namespace', viewNamespace);
         dispatch(
-          globalAppRegistryEmit('compass:aggregations:update-view', {
-            numStages: viewPipeline.length,
-          })
+          globalAppRegistryEmit(
+            'aggregations-open-view-after-update',
```

Comment:
This follows the same logic all the other places follow when opening namespace: emit a more specific event with only namespace, allow instance-store to get all required meta, then emit an event that will actually open a namespace, that way we don't spread meta composition all over the place